src/bindings/js/node/tests/unit/basic.test.js

```diff
@@ -33,6 +34,24 @@ describe('ov basic tests.', () => {
     assert.ok(devices.includes('CPU'));
   });
 
+  describe('ov.saveModel()', () => {
+    it('saveModel(model:Model, path:string, compressToFp16:bool=true)', () => {
+      const xmlPath = `${outDir}/${modelLike[0].getName()}_fp16.xml`;
+      assert.doesNotThrow(() => ov.saveModel(modelLike[0], xmlPath));
```

Comment:
Do you mean? (`it('saveModel(model:Model, path:string, compressToFp16:bool=true)',`)
```suggestion
      assert.doesNotThrow(() => ov.saveModel(modelLike[0], xmlPath, true));
```